gnomad_qc/v4/resources/sample_qc.py

```diff
@@ -846,3 +846,16 @@ def finalized_outlier_filtering(test: bool = False) -> VersionedTableResource:
         for version in VERSIONS
     },
 )
+
+
+# Sample QC meta HT global and row annotation definitions.
+def get_sample_qc_field_def_json_path(version: str) -> str:
+    """
+    Get path to JSON file containing sample QC metadata HT field definitions.
+
+    :param version: gnomAD version.
+    :return: Path to sample QC field definitions JSON.
+    """
+    return (
+        f"{get_sample_qc_root(version, data_type='joint')}/additional_resources/gnomad.joint.{version}.sample_qc_field_definitions.json"
```

Comment:
I think we want it to be exomes right? at some point I do want a joint HT too, but this one is only exomes

Reply:
Yeah that makes sense, I was going by the `joint` in the filename `gnomad.joint.sample_qc...` I'll change this bit to exomes too.